Packs/CloudShare/ReleaseNotes/1_0_1.md

```diff
@@ -1,4 +1,4 @@
 
 #### Integrations
 ##### CloudShare (Beta)
-- Fixed an issue where the **cloudshare-delete-class-environemnts** command was incorrectly handled in the code.
+- Fixed an issue where the **cloudshare-delete-class-environments** command was incorrectly handled in the code.
```

Comment:
I don't think you can edit a pre-existing RN.

Reply:
I think this is true